cabal.project

```diff
@@ -20,18 +20,32 @@ package hydra-plutus
 -- Always show detailed output for tests
 test-show-details: direct
 
+-- This is the incantation which seems to be pleasing the cabal gods, given the
+-- current 'index-state' and versions of 'plutus' and 'cardano-node'. Possible
+-- explanation why this is necessary: we need a more recent 'index-state'
+-- (because of plutus) than cardano-node et al and some of their dependencies do
+-- not have behave well.
+constraints:
+  attoparsec == 0.13.2.5,
+  cryptonite == 0.27,
+  dlist == 0.8.0.8,
+  dns == 3.0.4,
+  semialign == 1.1.0.1,
+  lens == 4.19.2,
+  dependent-sum == 0.7.1.0
+
 allow-newer:
-  -- 'iohk-monitoring' -> 'ekg' -> 'snap-server' which has a < 0.14 upper bound for attoparsec
-  snap-server:attoparsec
+  some
```

Comment:
Initially I thought this was a keyword 😂

Reply:
This should be merged with the other `allow-newer` section down below?

Reply:
Kept it separate as the one below is verbatim from plutus and it seems to be merged by cabal